config.json.sample

```diff
@@ -0,0 +1,5 @@
+{
+  "debug": false,
+  "featureManagerIsFullZoneProvisioningEnabled": true,
+  "locale": "en"
+}
```

Comment:
I'd suggest putting everything deleted in config.js to config.json.sample. This file will never be used in the frontend but it'll act as a documentation of configurable settings in Frontend.